cookbooks/pelias/attributes/external.rb

```diff
@@ -3,7 +3,6 @@
 
 # nodejs
 default[:nodejs][:version]                      = '4.4.7'
-default[:nodejs][:checksum_linux_x64]           = '4e1578efc2a2cc67651413a05ccc4c5d43f6b4329c599901c556f24d93cd0508'
```

Comment:
FWIW, I replaced these lines with:
```
diff --git a/cookbooks/pelias/attributes/external.rb b/cookbooks/pelias/attributes/external.rb
index fbc98be..e5e6e0f 100644
--- a/cookbooks/pelias/attributes/external.rb
+++ b/cookbooks/pelias/attributes/external.rb
@@ -2,11 +2,11 @@
 #
 
 # nodejs
-default[:nodejs][:version]                      = '0.12.2'
-default[:nodejs][:checksum_linux_x64]           = '4e1578efc2a2cc67651413a05ccc4c5d43f6b4329c599901c556f24d93cd0508'
+default['nodejs']['version'] = '0.12.2'
+default['nodejs']['binary']['checksum'] = '4e1578efc2a2cc67651413a05ccc4c5d43f6b4329c599901c556f24d93cd0508'
```
and it seems to work (at least the nodejs part succeeds). I do not know if not specifying the arch is a problem in vagrant context, could it setup a 32-bit ubuntu? If not, it seems better to keep the checksum don't you think?